recipes/r-semaphore/meta.yaml

```
@@ -0,0 +1,92 @@
+{% set version = '1.0.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-semaphore
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/semaphore_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/semaphore/semaphore_{{ version }}.tar.gz
+  sha256: 017443979225724eeb7060bf8c6effa0b89abef2040acfdff83bf1fd0f0675e6
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ stdlib('c') }}
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-bh
+    - r-rcpp
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-bh
```

```suggestion
    #- r-bh
```

`r-bh` is quite big and only needed during compile-time.